.github/workflows/ci.yml

```diff
@@ -5,12 +5,23 @@ on: [push, pull_request]
 jobs:
   tests:
     runs-on: ubuntu-20.04
+    container:
+      image: "ubuntu:20.04"
     steps:
       - uses: actions/checkout@v2
       - name: Install required dependencies
         run: |
-          sudo apt install --yes npm
+          apt-get update
+          apt-get install -y sudo
```

Comment:
@parhamsaremi please replace all uses of `apt-get` with `apt`

Reply:
this is not done (same problem in the other yml file)

Reply:
sorry, it was done when I commented done. but I needed to use an older version and I forgot to re-apply these.